### PR TITLE
revise an expression in "Highly Concurrent Write Best Practices"

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)
Revise an expression in "Highly Concurrent Write Best Practices"

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/8140
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by tiancaiamao — APPROVED** [review]

File: best-practices/high-concurrency-best-practices.md [hunk 1]
```diff
@@ -39,11 +39,13 @@ TiDB splits data into Regions, each representing a range of data with a size lim
 
 ![TiDB Data Overview](/media/best-practices/tidb-data-overview.png)
 
-In theory, by the virtue of this architecture, TiDB can linearly scale its read and write capacities and make full use of the distributed resources so long as there is no `AUTO_INCREMENT` primary key in the write scenario, or there is no monotonically increasing index. From this point of view, TiDB is especially suitable for the highly-concurrent and write-intensive scenario. However, the actual situation often differs from the theoretical assumption.
+In theory, by the virtue of its architecture, TiDB can not only linearly scale its read and write capacities but also make full use of the distributed resources as long as there is no write hotspot in the application. From this point of view, TiDB is especially suitable for the high-concurrent and write-intensive scenario.
```
Comment (en-jin19) [inline best-practices/high-concurrency-best-practices.md:42]:
```suggestion
In theory, if the application has no write hotspot, TiDB, by the virtue of its architecture, can not only linearly scale its read and write capacities but also make full use of the distributed resources. From this point of view, TiDB is especially suitable for the high-concurrent and write-intensive scenario.
```

File: best-practices/high-concurrency-best-practices.md [hunk 2]
```diff
@@ -39,11 +39,13 @@ TiDB splits data into Regions, each representing a range of data with a size lim
...
+
+However, the actual situation often differs from the theoretical assumption.
 
 > **Note:**
 >
-> No `AUTO_INCREMENT` primary key in the write scenario or no monotonically increasing index means no write hotspot in the application.
+> No write hotspot in the application means no `AUTO_INCREMENT` primary key or no monotonically increasing index in the write scenario.
```
Comment (en-jin19) [inline best-practices/high-concurrency-best-practices.md:48]:
```suggestion
> No write hotspot in the application means the write scenario does not have `AUTO_INCREMENT` primary key or monotonically increasing index.
```

File: best-practices/high-concurrency-best-practices.md [hunk 3]
```diff
@@ -39,11 +39,13 @@ TiDB splits data into Regions, each representing a range of data with a size lim
 
 ![TiDB Data Overview](/media/best-practices/tidb-data-overview.png)
 
-In theory, by the virtue of this architecture, TiDB can linearly scale its read and write capacities and make full use of the distributed resources so long as there is no `AUTO_INCREMENT` primary key in the write scenario, or there is no monotonically increasing index. From this point of view, TiDB is especially suitable for the highly-concurrent and write-intensive scenario. However, the actual situation often differs from the theoretical assumption.
+In theory, if the application has no write hotspot, TiDB, by the virtue of its architecture, can not only linearly scale its read and write capacities but also make full use of the distributed resources. From this point of view, TiDB is especially suitable for the high-concurrent and write-intensive scenario.
```
Comment (shichun-0415) [inline best-practices/high-concurrency-best-practices.md:42]:
```suggestion
In theory, if an application has no write hotspot, TiDB, by the virtue of its architecture, can not only linearly scale its read and write capacities, but also make full use of the distributed resources. From this point of view, TiDB is especially suitable for the high-concurrent and write-intensive scenario.
```

File: best-practices/high-concurrency-best-practices.md [hunk 4]
```diff
@@ -39,11 +39,13 @@ TiDB splits data into Regions, each representing a range of data with a size lim
...
+
+However, the actual situation often differs from the theoretical assumption.
 
 > **Note:**
 >
-> No `AUTO_INCREMENT` primary key in the write scenario or no monotonically increasing index means no write hotspot in the application.
+> No write hotspot in the application means the write scenario does not have `AUTO_INCREMENT` primary key or monotonically increasing index.
```
Comment (shichun-0415) [inline best-practices/high-concurrency-best-practices.md:48]:
```suggestion
> No write hotspot in an application means the write scenario does not have any `AUTO_INCREMENT` primary key or monotonically increasing index.
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

**Pull request merged**